_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-midnight
```

Comment:
I don't think that you want to commit that.

Reply:
You're right.